src/csharp/Grpc.Core/Internal/PlatformApis.cs

```diff
@@ -53,7 +53,7 @@ internal static class PlatformApis
 
         static PlatformApis()
         {
-#if NETSTANDARD1_5
+#if !NET45
```

Comment:
If someone were to run on net451, but use Grpc net45 target, would this break? E.g., something like: 
`frameworks: { net451: { imports: net45 } }`, (If net451 doesn't automatically import net45?)